tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -246,15 +246,28 @@ def test_show_interfaces_status(self, setup_config_mode, sample_intf):
         test_intf = sample_intf[mode]
         interface = sample_intf['default']
         interface_alias = sample_intf['alias']
-        regex_int = re.compile(r'(\S+)\s+[\d,N\/A]+\s+(\w+)\s+(\d+)\s+([\w\/]+)\s+(\w+)\s+(\w+)\s+(\w+)')
+        regex_int_old = re.compile(r"""
+                                   (\S+)\s+[\d,N\/A]+\s+(\w+)\s+(\d+)\s+
+                                   ([\w\/]+)\s+(\w+)\s+(\w+)\s+(\w+)
+                                   """, re.VERBOSE)
+        regex_int_fec = re.compile(r"""
+                                   (\S+)\s+[\d,N\/A]+\s+(\w+)\s+(\d+)\s+
+                                   (rs|N\/A)\s+([\w\/]+)\s+
+                                   (\w+)\s+(\w+)\s+(\w+)
+                                   """, re.VERBOSE)
```

Comment:
Can we replace this section with code with the module?
https://github.com/Azure/sonic-mgmt/blob/master/ansible/library/show_interface.py

Reply:
Yes, but have to refactor ansible module `show_interface` a little bit to support env variable `SONIC_CLI_IFACE_MODE`, I will take a try.

Reply:
Updated, pls take a look.